tests/unit/test_memory_cache.py

```diff
@@ -0,0 +1,61 @@
+import csp_billing_adapter
+from csp_billing_adapter.memory_cache import memory_cache
+
+class TestMemoryCache:
```

Comment:
For my tests, I was getting a warning about "Method could be a function" since I was not using self. I switched to a non Class based test structure to resolve this. If you are not using self, should we be consistent and just make these functions?